services/dependabot/dependabot.service.js

```diff
@@ -1,66 +1,57 @@
 'use strict'
 
-const LegacyService = require('../legacy-service')
-const {
-  makeBadgeData: getBadgeData,
-  makeLogo: getLogo,
-} = require('../../lib/badge-data')
-const { checkErrorResponse } = require('../../lib/error-helper')
+const BaseJsonService = require('../base-json')
+const Joi = require('joi')
+
+const schema = Joi.object({
+  status: Joi.string().required(),
+  colour: Joi.string().required(),
+})
+
+module.exports = class DependabotSemverCompatibility extends BaseJsonService {
+  async fetch({ packageManager, dependencyName }) {
+    const url = `https://api.dependabot.com/badges/compatibility_score?package-manager=${packageManager}&dependency-name=${dependencyName}&version-scheme=semver`
```

Comment:
Oh yes. I had intended to factor this out so that we assemble the API and link querystring in the same way. Then I got bogged down in the default logo stuff and overlooked it.